ods_ci/tasks/Resources/Provisioning/Hive/gpu-provision.robot

```diff
@@ -15,6 +24,46 @@ Delete GPU Node In Self Managed AWS Cluster
     ${gpu_machineset} =    Oc Get    kind=MachineSet    label_selector=gpu-machineset=true
     Run And Return Rc    oc scale --replicas=0 machineset/${gpu_machineset[0]['metadata']['name']} -n openshift-machine-api
 
+Create GPU Nodes
+    [Documentation]    Create GPU nodes in both managed and self-managed clusters.
+    ...    Parameters: cluster_type (managed/self-managed), instance_type (g4dn.xlarge),
+    ...    provider (AWS/GCP/AZURE/IBM), gpu_node_count (1), cluster_name (for managed).
+    [Arguments]    ${cluster_type}=self-managed
+    ...    ${instance_type}=g4dn.xlarge
+    ...    ${provider}=AWS
+    ...    ${gpu_node_count}=1
+    ...    ${cluster_name}=${EMPTY}
+    Set Log Level    Info
+
+    IF    '${cluster_type}' == 'managed'
+        # For managed clusters, use OCM to add machine pool
+        IF    '${cluster_name}' == '${EMPTY}'
+            Fail    cluster_name is required for managed clusters
+        END
+        ${result} =    Run Process    python    utils/scripts/ocm/ocm.py    add_machine_pool
+        ...    --cluster-name    ${cluster_name}
+        ...    --instance-type    ${instance_type}
+        ...    --worker-node-count    ${gpu_node_count}
+        ...    --pool-name    gpu-pool-${gpu_node_count}
```

Comment:
pool name can collide. Add a unique suffix - timestamp or can it be re-used if exists.